tasks/main.yml

```diff
@@ -29,6 +29,16 @@
     - common.py
     - common.sh
 
+- name: Copy check scripts dependencies
+  copy:
+    src: "{{ item }}"
+    dest: "{{ icinga2_scripts_path }}"
+    mode: 0755
+    owner: root
+    group: root
+  with_items:
+    - tinkerforge
```

Comment:
Maybe `when: needs_tinkerforge`. This way, we don't push the file to ALL servers